pkg/cmd/roachprod/install/expander.go

```diff
@@ -100,26 +114,32 @@ func (e *expander) maybeExpandUIPort(c *SyncedCluster, s string) (string, bool)
 		}
 	}
 
-	return e.maybeExpandMap(c, e.uiPorts, m[1])
+	s, err := e.maybeExpandMap(c, e.uiPorts, m[1])
+	return s, err == nil, err
 }
 
-func (e *expander) maybeExpandStoreDir(c *SyncedCluster, s string) (string, bool) {
+func (e *expander) maybeExpandStoreDir(c *SyncedCluster, s string) (string, bool, error) {
```

Comment:
Ditto. Or maybe just a comment that says "look at the def of expander for info on the retvals"...